src/types/Errors.ts

```diff
@@ -0,0 +1,8 @@
+export enum Errors {
+  NO_TITLE = 'Title can\'t be empty',
+  ADD = 'Unable to add a todo',
+  DELETE = 'Unable to delete a todo',
+  UPDATE = 'Unable to update a todo',
+  LOAD = 'Unable to load a todos',
```

Comment:
Typo in the error message. It should be 'Unable to load todos' instead of 'Unable to load a todos'.